### PR TITLE
Doc: Improving vtex-io-documentation-edition-app.md

### PR DESCRIPTION
**Pull request by barbara-celi**
#### Types of changes
- [ ] New content (guides, endpoints, app documentation)
- [X] Improvement (make a documentation even better)
- [ ] Fix (fix a documentation error)
- [ ] Spelling and grammar accuracy (self-explanatory)

Differences between Edition Store app versions have been added to the documentation.

### CODE REVIEW

File: docs/vtex-io/Reference/concepts/vtex-io-documentation-edition-app.md [hunk 1]
```diff
@@ -7,16 +7,27 @@ createdAt: "2020-06-03T16:02:44.317Z"
...
 
 ## List of native Edition Apps
 
 The `vtex` account currently provides the following native Edition Apps:
 
-- Edition Store (`vtex.edition-store@5.x`) - installs all the necessary apps to develop a store with the [Store Framework](https://developers.vtex.com/docs/guides/getting-started-3).
 - Edition Business (`vtex.edition-business@0.x`) - installs all the necessary apps to build a store with the [Legacy CMS Portal](https://help.vtex.com/en/tracks/cms--2YcpgIljVaLVQYMzxQbc3z/6OCY6S9tqBXPD5mgpbBInC).
+- Edition Store (`vtex.edition-store@5.x`) - installs all the necessary apps to develop a store with the [Store Framework](https://developers.vtex.com/docs/guides/getting-started-3).
+
+There are currently 4 available versions of the Edition Store. Next, you can better understand the differences between them:
```
Comment (julia-rabello) [inline docs/vtex-io/Reference/concepts/vtex-io-documentation-edition-app.md:21]:
```suggestion
There are currently four available versions of the Edition Store. Next, you can better understand the differences between them:
```

File: docs/vtex-io/Reference/concepts/vtex-io-documentation-edition-app.md [hunk 2]
```diff
@@ -7,16 +7,27 @@ createdAt: "2020-06-03T16:02:44.317Z"
...
+
+| Versions | What is new? |
+| --- | --- |
+|  2.x  | Standard version  |
+|  3.x  | `vtex.store-sitemap@2.x` app declared in the manifest’s [dependencies](https://developers.vtex.com/docs/guides/vtex-io-documentation-dependencies)  |
+|  4.x  | Majors of `vtex.checkout` app, including both 1.x and 2.x versions, declared in the manifest’s dependencies  |
+|  5.x  | [Search apps](https://developers.vtex.com/docs/apps/vtex.search) installed  |
+
+>⚠️ If you are [migrating from Legacy CMS Portal to Store Framework](https://developers.vtex.com/docs/guides/vtex-io-documentation-migrating-storefront-from-legacy-to-io) or are using the 2.x version and want to update, it is not necessary to install the 5.x version **if your store already has the Search apps installed**. You can install 3.x or 4.x versions, depending on your case. If you have doubts about which version to set in your store, please [open a ticket](https://help-tickets.vtex.com/smartlink/sso/login/zendesk) to the VTEX Support team.
```
Comment (julia-rabello) [inline docs/vtex-io/Reference/concepts/vtex-io-documentation-edition-app.md:30]:
```suggestion
>⚠️ If you are [migrating from Legacy CMS Portal to Store Framework](https://developers.vtex.com/docs/guides/vtex-io-documentation-migrating-storefront-from-legacy-to-io) or are using the 2.x version and want to update, it is not necessary to install the 5.x version **if your store already has the Search apps installed**. You can install 3.x or 4.x versions, depending on your case. If you are not sure which version to set in your store, please [open a ticket](https://help-tickets.vtex.com/smartlink/sso/login/zendesk) to the VTEX Support team.
```

File: docs/vtex-io/Reference/concepts/vtex-io-documentation-edition-app.md [hunk 3]
```diff
@@ -38,4 +49,6 @@ All Edition apps must depend, either directly or indirectly, on either `vtex.edi
 
 Apps included as [`dependencies`](https://developers.vtex.com/docs/guides/vtex-io-documentation-dependencies) within an Edition App are considered inherited apps. Moreover, all apps bundled within an Edition App must share the same [`vendor`](https://developers.vtex.com/docs/guides/vtex-io-documentation-manifest) as the Edition App itself.
 
-It's important to note that Sponsored Accounts cannot modify apps and configurations installed through an Edition App. Only the Sponsor Account has the authority to make changes by releasing a new version of its Edition App and requesting a reinstallation.
+It is important to note that Sponsored Accounts cannot modify apps and configurations installed through an Edition App. Only the Sponsor Account has the authority to make changes by releasing a new version of its Edition App and requesting a reinstallation.
```
Comment (julia-rabello) [inline docs/vtex-io/Reference/concepts/vtex-io-documentation-edition-app.md:52]:
```suggestion
> ℹ It is important to note that Sponsored Accounts cannot modify apps and configurations installed through an Edition App. Only the Sponsor Account has the authority to make changes by releasing a new version of its Edition App and requesting a reinstallation.
```

File: docs/vtex-io/Reference/concepts/vtex-io-documentation-edition-app.md [hunk 4]
```diff
@@ -38,4 +49,6 @@ All Edition apps must depend, either directly or indirectly, on either `vtex.edi
...
+
```
Comment (julia-rabello) [inline docs/vtex-io/Reference/concepts/vtex-io-documentation-edition-app.md:53]:
```suggestion
```

**Review by julia-rabello — CHANGES_REQUESTED** [review]

File: docs/vtex-io/Reference/concepts/vtex-io-documentation-edition-app.md [hunk 5]
```diff
@@ -7,16 +7,27 @@ createdAt: "2020-06-03T16:02:44.317Z"
...
+- Edition Store (`vtex.edition-store@5.x`) - installs all the necessary apps to develop a store with the [Store Framework](https://developers.vtex.com/docs/guides/getting-started-3).
+
+There are currently four available versions of the Edition Store. Next, you can better understand the differences between them:
+
+| Versions | What is new? |
+| --- | --- |
+|  2.x  | Standard version  |
+|  3.x  | `vtex.store-sitemap@2.x` app declared in the manifest’s [dependencies](https://developers.vtex.com/docs/guides/vtex-io-documentation-dependencies)  |
+|  4.x  | Majors of `vtex.checkout` app, including both 1.x and 2.x versions, declared in the manifest’s dependencies  |
+|  5.x  | [Search apps](https://developers.vtex.com/docs/apps/vtex.search) installed  |
```
Comment (carolinamenezes) [inline docs/vtex-io/Reference/concepts/vtex-io-documentation-edition-app.md:28]: Acho que seria interessante listar os apps e marcar com  ❌  ou ✅ . Tipo assim:

|Version| vtex.store-sitemap | vtex.checkout | [Intelligent Search apps](https://developers.vtex.com/docs/apps/vtex.search)|
|-----|----------|--------|-----------|
|`2.x`| ❌ | ❌ | ❌ |
|`3.x`| ✅ | ❌   | ❌ |
|`4.x`| ❌ | ✅  |  ❌ |
|`3.x`| ✅ | ✅  | ✅ |

Comment (carolinamenezes) [reply docs/vtex-io/Reference/concepts/vtex-io-documentation-edition-app.md:28]: Fica mais visual e claro o que cada versão tem/não tem

Comment (barbara-celi) [reply docs/vtex-io/Reference/concepts/vtex-io-documentation-edition-app.md:28]: Fiz o ajuste, ficou assim:

There are currently four available versions of the Edition Store. Next, you can better understand their differences regarding the apps installed by default in the manifest’s [dependencies](https://developers.vtex.com/docs/guides/vtex-io-documentation-dependencies).

| Version	| [vtex.store-sitemap](https://developers.vtex.com/docs/apps/vtex.store-sitemap)	| vtex.checkout (including 1.x and 2.x  versions) | [vtex.search](https://developers.vtex.com/docs/apps/vtex.search)
|   :---:   |   :---:   |   :---:   |   :---:   |
| 2.x	| ❌ | ❌	| ❌ |
| 3.x	| ✅	| ❌	| ❌ |
| 4.x	| ✅	| ✅	| ❌ |
| 5.x	| ✅    | ✅	| ✅ |

**Pull request merged**